### PR TITLE
v9.3.0 - fixes spacing & zIndex references

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Fixed
- `spacing` & `zIndex` references.

### CODE REVIEW

**Review by jamieomaguire — DISMISSED** [review]

File: src/scss/base/_layout.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 @use '../settings/variables' as v;
 @use '../settings/include-media' as *;
 @use '../tools/mixins/type';
+@use '../tools/functions/spacing' as spacing;
```
Comment (DamianMullins) [inline src/scss/base/_layout.scss:5]: Minor thing, you can leave the `as spacing` part off and it'll use `spacing` as the module name, like does `type` above.

```suggestion
@use '../tools/functions/spacing';
```

File: src/scss/components/optional/_cookie-warning.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
 @use '@justeat/pie-design-tokens/dist/jet' as dt;
 @use '../../tools/mixins/type';
+@use '../../tools/functions/spacing' as spacing;
+@use '../../tools/functions/zIndex' as zIndex;
```
Comment (DamianMullins) [inline src/scss/components/optional/_cookie-warning.scss:4]: Same as my last comment, you don't need to specify the `as` part.

**Review by DamianMullins — COMMENTED** [review]
I just checked this branch locally and it looks like there are still some modules which need updating.

 * `_alerts.scss`
 * `_listings-skeleton.scss`
 * `_order-card.scss`
 * `_buttons.scss`
 * `_form-toggle.scss`

**Review by DamianMullins — APPROVED** [review]

File: src/scss/base/_layout.scss (hunk 1 above)
Comment (jamieomaguire) [inline src/scss/base/_layout.scss:5]: can probably remove the 'as' from this too :)

File: src/scss/base/_typography.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @use '../settings/variables' as v;
 @use '@justeat/pie-design-tokens/dist/jet' as dt;
 @use '../tools/mixins/type';
+@use '../tools/functions/spacing' as spacing;
```
Comment (jamieomaguire) [inline src/scss/base/_typography.scss:4]: and here

File: src/scss/components/_alerts.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
 @use '@justeat/pie-design-tokens/dist/jet' as dt;
 @use '../tools/mixins/alerts';
+@use '../tools/functions/spacing' as spacing;
```
Comment (jamieomaguire) [inline src/scss/components/_alerts.scss:3]: and here

File: src/scss/components/_breadcrumbs.scss [hunk 5]
```diff
@@ -1,6 +1,7 @@
 @use '@justeat/pie-design-tokens/dist/jet' as dt;
 @use '../settings/include-media' as *;
 @use '../tools/mixins/type';
+@use '../tools/functions/spacing' as spacing;
```
Comment (jamieomaguire) [inline src/scss/components/_breadcrumbs.scss:4]: and here

File: src/scss/components/_cards.scss [hunk 6]
```diff
@@ -1,6 +1,7 @@
 @use '@justeat/pie-design-tokens/dist/jet' as dt;
 @use '../settings/include-media' as *;
 @use '../tools/mixins/truncate';
+@use '../tools/functions/spacing' as spacing;
```
Comment (jamieomaguire) [inline src/scss/components/_cards.scss:4]: and here

File: src/scss/base/_layout.scss (hunk 1 above)
Comment (kevinrodrigues) [reply src/scss/base/_layout.scss:5]: Will be creating a ticket to tidy this all up. Once we've agreed on a convention.

**Review by jamieomaguire — APPROVED** [review]

File: src/scss/base/_layout.scss (hunk 1 above)
Comment (ashleynolan) [reply src/scss/base/_layout.scss:5]: I'm not 100% sure if it'll work, but would it be possible to potentially do:

```scss
@use '../tools/functions/index as functions;
```

and then reference `functions.spacing()`

I think it should work with that syntax and make the namespaces a bit cleaner?

Comment (kevinrodrigues) [reply src/scss/base/_layout.scss:5]: I can give this a try.

Comment (kevinrodrigues) [reply src/scss/base/_layout.scss:5]: This is now fixed here https://github.com/justeat/fozzie/pull/401. I took the function route it worked well 👍

**Pull request merged**